### PR TITLE
zh-translation: /news/releases/1.3.x/announcing-1.3.5/index.md #1580

### PR DESCRIPTION
**Pull request by gorda**
Ref: [ServiceMesher#1580](https://github.com/servicemesher/istio-official-translation/issues/1580)

### CODE REVIEW

File: content/zh/news/releases/1.3.x/announcing-1.3.5/index.md [hunk 1]
```diff
@@ -1,34 +1,35 @@
...
-- **Fixed** Pilot to no longer crash when an invalid Envoy configuration is generated. ([Issue 17266](https://github.com/istio/istio/issues/17266))
-- **Fixed** an issue with the `destination_service_name` label not getting populated for TCP metrics related to BlackHole/Passthrough clusters. ([Issue 17271](https://github.com/istio/istio/issues/17271))
-- **Fixed** an issue with telemetry not reporting metrics for BlackHole/Passthrough clusters when fall through filter chains were invoked. This occurred when explicit `ServiceEntries` were configured for external services. ([Issue 17759](https://github.com/istio/istio/issues/17759))
+- **修复** TCP headless 服务的 Envoy 监听器配置。（[Issue #17748](https://github.com/istio/istio/issues/17748)）
+- **修复** 即使将 deployment 缩放到 0 个副本，过时的 endpoint 也会保留的问题。（[Issue #14436](https://github.com/istio/istio/issues/14336)）
+- **修复** 生成无效的 Envoy 配置时，Pilot 不会再崩溃。（[Issue 17266](https://github.com/istio/istio/issues/17266)）
+- **修复** 没有为与 BlackHole/Passthrough 群集相关的 TCP 度量填充 `destination_service_name` 标签的问题。（[Issue 17271](https://github.com/istio/istio/issues/17271)）
```
Comment (yuxiaobo96) [inline content/zh/news/releases/1.3.x/announcing-1.3.5/index.md]:
```suggestion
- **修复** 没有为与 BlackHole/Passthrough 集群相关的 TCP 指标填充 `destination_service_name` 标签的问题。（[Issue 17271](https://github.com/istio/istio/issues/17271)）
```

File: content/zh/news/releases/1.3.x/announcing-1.3.5/index.md [hunk 2]
```diff
@@ -1,34 +1,35 @@
...
+- **修复** 调用遥测过滤器链时遥测不报告 BlackHole/Passthrough 群集指标的问题。该问题会在显示的为外部服务配置了 `ServiceEntries` 时发生的。
```
Comment (yuxiaobo96) [inline content/zh/news/releases/1.3.x/announcing-1.3.5/index.md]:
```suggestion
- **修复** 调用遥测过滤器链时遥测不报告 BlackHole/Passthrough 集群指标的问题。该问题会在为外部服务配置显示的 `ServiceEntries` 时发生的。
```

Comment (gorda) [reply content/zh/news/releases/1.3.x/announcing-1.3.5/index.md]: @yuxiaobo96 感谢细心 review，相关问题已修改

**Comment by yuxiaobo96** [comment]
@rootsongjc 
已修改，可以合并

**Pull request merged**